### PR TITLE
feat: add open app route

### PR DESCRIPTION
**Pull request by pengx17**
Fix electron login (email) based on https://github.com/toeverything/AFFiNE/issues/3944
also fixes https://github.com/toeverything/AFFiNE/issues/3831

### CODE REVIEW

File: apps/electron/src/main/deep-link.ts [hunk 1]
```diff
@@ -1,16 +1,30 @@
 import type { App } from 'electron';
+import path from 'path';
```
Comment (himself65) [inline apps/electron/src/main/deep-link.ts]: node:path

**Review by himself65 — CHANGES_REQUESTED** [review]

File: apps/core/src/components/affine/desktop-login-modal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Modal, ModalWrapper } from '@affine/component';
+
+export const DesktopLoginModal = ({ open }: { open: boolean }) => {
+  return (
+    <Modal open={open}>
+      <ModalWrapper width={560} style={{ padding: '10px' }}>
+        Logging in ...
```
Comment (Garfield550) [inline apps/core/src/components/affine/desktop-login-modal/index.tsx:7]: i18n please.

**Review by Garfield550 — CHANGES_REQUESTED** [review]

File: apps/core/src/components/affine/desktop-login-modal/index.tsx (hunk 2 above)
Comment (pengx17) [reply apps/core/src/components/affine/desktop-login-modal/index.tsx:7]: Will do. This is still wip :)

**Review by himself65 — COMMENTED** [review]
rebased to master branch

**Review by himself65 — APPROVED** [review]

**Pull request merged**